src/components/MetaDecorator.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Helmet, HelmetProvider } from 'react-helmet-async';
+
+function MetaDecorator({ title, description }) {
+    if (!title || !description) {
+        title = 'Open Sauced';
+        description = 'The path to your next open-source contribution.';
+    }
+    return (
+        <HelmetProvider>
+            <Helmet>
+                <title>{title}</title>
+                <meta name="description" content={description} />
+
+            //Open Graph Protocol Meta
+                <meta property="og:title" content={title} />
+                <meta property="og:description" content={description} />
+                <meta property="og:image" content="https://i.ibb.co/my2VqXw/sauced-social-card.png" />
```

Comment:
Can we replace the URL here to "https://repository-images.githubusercontent.com/71359796/179eca00-061e-11eb-9b57-856a179db1a1" it matches the uploaded social for the repo